docs/src/index.md

```diff
@@ -24,6 +24,51 @@ or coupling components (using MPI) that are running on different processes.
 |:--:|
 | *ClimaCoupler.jl allows for independent development of interchangeable component models.* |
 
+
+## Current AMIP components
+
+### Atmosphere
+
+Dynamical core:
+- Equation: non-hydrostatic and fully compressible
+- Prognostic variables: Density, velocity components, total energy, total specific humidity
+- Spatial discretization: Spectral element in the horizontal, finite difference in the vertical, cubed sphere
+- Time stepping: Implicit-explicit additive Runge–Kutta method
+
+Radiation: A scheme based on RRTM for General circulation model applications—Parallel (RRTMGP)
+
+Convection and turbulence: Diagnostic Eddy-diffusivity Mass-Flux scheme with prognostic turbulent kinetic energy
+
+Microphysics: 0-moment scheme, where cloud condensate is removed with a constant timescale
+
+Surface fluxes: A scheme based on Monin-Obukhov similarity theory, with constant roughness lengths over land and ocean
+
+Orographic gravity wave drag: None
+
+Non-orographic gravity wave drag: None
+
+Aerosols and chemistry: None
+
+### Land
+
+Dynamical core:
+- Equation: Following Manabe bucket hydrology scheme
+- Prognostic variables: Temperature, water content, snow water content
+- Spatial discretization: Finite difference, single column
```

Comment:
is single column correct? I thought for global runs we have many columns over the sphere

Reply:
Ah, yes, I meant there is no horizontal transport between each column. I'll think about a better description.